packages/cli/src/commands/show.js

```diff
@@ -1,10 +1,14 @@
 /* global process */
 import os from 'os';
 import { E } from '@endo/far';
-import { withEndoParty } from '../context.js';
+import { withEndoBootstrap } from '../context.js';
 
-export const show = async ({ cancel, cancelled, sockPath, name, partyNames }) =>
-  withEndoParty(partyNames, { os, process }, async ({ party }) => {
-    const pet = await E(party).lookup(name);
+export const show = async ({ cancel, cancelled, sockPath, namePath }) =>
+  withEndoBootstrap({ os, process }, async ({ bootstrap }) => {
+    const defaultHostFormulaIdentifier = 'host';
+    const pet = await E(bootstrap).lookupFrom(
```

Comment:
Ah, I see. In my head, `lookupFrom` is equivalent to `E(E(host).locate(identifier)).lookup(path)` but consolidated. The consolidation might be important since we might want to locate and lookup locally in a single event.

I think we should move the method to Host in order to stay one step away from folding the Endo Bootstrap object into Host objects. They are equally powerful since you can’t get one without the other and there’s little reason to keep them separate.

I’m not attached to the `locate(nonce) ~=> identifier` function name, and I think I at least temporarily need the name of this method to be explicitly `lookupFromFormulaNumber`, with goal of reducing that to `lookupFromIdentifier` when we no longer need to distinguish formula identifier with type from formula number.